geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -1979,7 +1978,7 @@ public void memberDeparted(DistributionManager distributionManager,
     if (!getSender().equals(id)) {
       return;
     }
-    this.dm.removeMembershipListener(this);
+    getDistributionManager().removeMembershipListener(this);
```

Comment:
it would be better to just use the "distributionManager" parameter. At some point we may find that this class no longer needs to keep a reference to the dm in an instance field.

Reply:
Will address this concern with additional dunit test.